packages/admin/src/models/contract.ts

```diff
@@ -7,4 +7,5 @@ export interface Contract {
   address: Address;
   name: string;
   abi?: string;
+  natSpec?: string;
```

Comment:
It looks like this interface is used when creating a contract, so I think this implies that `natSpec` can be provided. Should probably split this into multiple types to handle the difference between a request and a response